src/stats.py

```diff
@@ -128,7 +128,29 @@ def de_emojify(self, text):
 
         :param text: Text that contains emoji
         """
-        regrex_pattern = re.compile(pattern="[\u2069\u2066]+", flags=re.UNICODE)
+        regrex_pattern = re.compile(pattern="["
```

Comment:
Since in the next line the ```demoji.replace(text, " ")```  is responsible for deleting all the emojis, we do not need to delete each emoji separately.

You can find the emojis that method ```demoji.replace()``` handles in the [package document](https://github.com/bsolomon1124/demoji/blob/master/README.md):
https://unicode.org/Public/emoji/14.0/emoji-sequences.txt